docs/frontend/styleguide.md

```diff
@@ -0,0 +1,45 @@
+# Shopsys Framework styleguide
+
+## Documentation
+styleguide - tool to make creating and maintaining styleguides easy for Shopsys Framework designs.
+Contains all html elements necessary to create new design.
+
+## Installation to running SSFW project
+You need to have Shopsys Framework installed in developer mode according to our installation guide
+
+1) run `php phing grunt` so you have all your styles compiled
+2) open `your-project-url/_styleguide` file in browser to see your styleguide
+
+### How to add new information
```

Comment:
What information? This should be described in better way

Reply:
Remaked.